app/routes/__main/subscription/__main/allSubscription/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const allSubscription = () => (
+   <section className="screen create tabContainer">
+      <div>
+         <h1 style={{color: "black"}}>hello</h1>
+      </div>
```

Comment:
This should be another component imported from `app/components/PurchaseSubscription.tsx`